src/content/blog/2024-plans.md

```diff
@@ -0,0 +1,47 @@
+pubDate: 2024-03-03
+#modifiedDate: 2024-03-03
+summary: Looking forward to 2024 with the DDEV Project
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/03/2024-ddev.png
+  alt: 2024 DDEV Plans
+  credit: 'Ideogram.ai: the words "2024" and "DDEV" next to each other'
+categories:
+  - Community
+---
+
+At the DDEV Advisory Group's annual review/planning meeting on March 6, 2024, we'll talk about plans and priorities for 2024. These have been discussed previously in the [DDEV 2024 Priorities discussion](https://github.com/orgs/ddev/discussions/5720) but in getting ready for the meeting it seems like a good time to propose them in a more cohesive fashion.
+
+## Community
+
+* **Continue the Contributor Training initiative**. In 2023 we had weekly [contributor training sessions](/blog/contributor-training) and they were a great way to put into practice our priority of making DDEV support and enhancement available to all community members.
+* **Continue outstanding user support** even with growth of our user base. In the past year the weekly users of DDEV about doubled, from about 5000 to 10,000 or more. This means increased support, and we want to continue our level of support in all the [support venues](https://ddev.readthedocs.io/en/stable/users/support/), but this likely means finding more community members to be first-line support instead of Randy and Stas often having to be there first (although we love helping!).
+* **Improve governance strategy for DDEV Foundation**.  Discuss governance with the community and consider raising the authority level of the advisory group, or  create another type of board. Make sure financial controls are obvious and adequate.
+
+## Sustainability and Finance
+
+* **Continue consistent marketing**, including blogs, clear fundraising efforts, newsletter. In 2023 we made marketing outreach a regular goal, as it had been neglected in the past. We want everybody to know that DDEV's future sustainability depends on them, and want them to be in the loop on what's going on. As a result of this, we now have a monthly [newsletter](/newsletter) and better communication around funding and supporting the project. We still have a long ways to go on this, and as always it competes for attention and time with other priorities.
+* **Recruit, train, and add a third paid co-maintainer**: Our two maintainers are 100% occupied with current priorities, and we should be able to grow funding (and maintainer capabilities) to add another co-maintainer.
+
+## Features
+
+* **Explicit [Upsun](https://upsun.com/)** by Platform.sh support, probably meaning an add-on for it, at least supporting the documented target CMSs.
+* **Improve xhprof usage**. The [ddev-xhgui](https://github.com/ddev/ddev-xhgui) add-on has demonstrated how much better the xhprof experience can be in DDEV. We would like to incorporate the ddev-xhgui work into DDEV core and make the great experience available to everyone without an add-on.
```

Comment:
I get page not found on this URL

Reply:
you probably meant https://github.com/tyler36/ddev-xhgui

Reply:
Thx for comments, fixed.